Makefile

```diff
@@ -168,6 +169,29 @@ host-agent-binary: $(RELEASE_DIR)
 		-o ./bin/$(notdir $(RELEASE_BINARY))-$(GOOS)-$(GOARCH) $(HOST_AGENT_DIR)
 
 
+##@Release
+
+$(RELEASE_DIR):
+	rm -rf $(RELEASE_DIR)
+	mkdir -p $(RELEASE_DIR)
+
+build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata-yaml build-host-agent-binary
```

Comment:
For `build-infra-yaml` target, we need to pass this variable:
`IMG=projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:rel-tag`
Otherwise, it will point to the default registry values - which is pointing to gcr.io and the provider won't be installed on the management cluster

Reply:
updated, Thanks.